root/static/scripts/release-editor/fields.js

```diff
@@ -1231,6 +1239,28 @@ class Release extends mbEntity.Release {
     }, []);
   }
 
+  // Returns a generator for iterating over all tracks.
+  allTracks() {
+    return (function* () {
+      for (const medium of this.mediums()) {
+        for (const track of medium.tracks()) {
+          yield track;
+        }
+      }
+    }).apply(this);
+  }
```

Comment:
Is there a reason having the star on `*allTracks()` wouldn't work? That should still return a fresh generator each time it's called IIUC.
```suggestion
  *allTracks() {
    for (const medium of this.mediums()) {
      for (const track of medium.tracks()) {
        yield track;
      }
    }
  }
```

Reply:
Thanks! I'm new to generator functions and had actually searched for e.g. `javascript method generator` to see if something like this was possible, but I apparently missed that the method syntax is actually documented near the bottom of <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*>. :-P

(eslint requires a space after the asterisk, which looks a bit weird, but oh well.)